docs/source/conf.py

```diff
@@ -8,15 +8,16 @@
 
 # -- Project information -----------------------------------------------------
 
-project = 'ASDF Standard'
+project = "ASDF Standard"
 
 import datetime
-copyright = f'{datetime.datetime.now().year}, Space Telescope Science Institute'
 
-author = 'Michael Droettboom, Erik Bray, et al'
+copyright = f"{datetime.datetime.now().year}, Space Telescope Science Institute"
+
+author = "Michael Droettboom, Erik Bray, et al"
```

Comment:
@nden @perrygreenfield should we change this to something like "The ASDF Developers"?

Reply:
That would be my preference.

Reply:
See 3b04a11